docs/architecture/modernize-desktop/example-migration-core.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Example of migrating to .NET Core 3.1
+description: Showing how to migrate a sample applications targeting  .NET Framework to .NET Core 3.1.
+ms.date: 09/16/2019
+---
+
+# Example of migrating to .NET Core 3.1
+
+In this chapter, we present practical guidelines to help you perform a migration of your existing application from .NET Framework to .NET Core.
+
+We present a well-structured process you can follow and the most important things to consider on each step.
+
+We then document a step-by-step migration process for a sample desktop application, both from WinForms and WPF versions.
+
+## Migration process overview
+
+The migration process consists of four sequential steps:
+
+1. **Preparation**: Understand the dependencies the project has to have an idea of what's ahead. In this step, you take the current project into a state that simplifies the startup point for the migration.
+
+2. **Migrate Project File:** .NET Core projects use the new SDK-style project format. Create a new project file with this format or update the one you have to use the SDK style.
+
+3. **Fix code and build:** Build the code in .NET Core addressing API-level differences between .NET Framework and .NET Core. If needed, update third-party packages to the ones that support .NET Core. WCF clients.
+
+4. **Run and test:** There might be differences that don't show up until the run time, so don't forget to run the application and test that everything works as expected.
+
+### Preparation
+
+#### Migrate packages.config file
+
+In a .NET Framework application, all references to external packages are declared in the *packages.config* file. In .NET Core, there's no longer the need to use the *packages.config* file. Instead, use the [PackageReference](../../core/project-sdk/msbuild-props.md#packagereference) property inside the project file to specify the NuGet packages for your app.
+
+So, you need to transition from one format to another. You can perform the update manually, taking the dependencies contained in the *packages.config* file and migrate them to the project file with the PackageReference format. Besides,
+you can leave Visual Studio do the work for you by right-clicking on the *packages.config* file and select the **Migrate packages.config to PackageReference** option.
+
+#### Verify every dependency compatibility in .NET Core
+
+Once you've migrated the package references, you must check each reference for compatibility. You can explore the dependencies of each NuGet package your application is using on [nuget.org](https://www.nuget.org/). If the package has
+.NET Standard dependencies, then it's going to work on .NET Core because .NET Core 3.1 [supports](../../standard/net-standard.md#net-implementation-support) all versions of .NET Standard. The following image shows the dependencies for
+the `Castle.Windsor` package:
```

Comment:
```suggestion
Once you've migrated the package references, you must check each reference for compatibility. You can explore the dependencies of each NuGet package your application is using on [nuget.org](https://www.nuget.org/). If the package has .NET Standard dependencies, then it's going to work on .NET Core because .NET Core 3.1 [supports](../../standard/net-standard.md#net-implementation-support) all versions of .NET Standard. The following image shows the dependencies for the `Castle.Windsor` package:
```